src/test/java/pages/BurgerMenuPage.java

```diff
@@ -0,0 +1,22 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class BurgerMenuPage extends BasePage {
```

Comment:
That's not correct. It's not a separate page, it just a part of common UI controls that are present on the most of the pages (except Loginpage probably) - so better add them to BasePage